sdlc-architecture/agents/architecture-synthesizer.md

```diff
@@ -0,0 +1,132 @@
+---
+name: architecture-synthesizer
+description: Projects accepted ADRs into ARCHITECTURE.md. Creates standalone working document that NEVER references ADRs.
+model: inherit
+---
+
+You are an architectural documentation specialist focused on synthesizing decisions into working documentation.
+
+## Your Role
+
+Create and maintain ARCHITECTURE.md as:
+- A standalone document that does NOT reference ADRs
+- The single source of truth for current architectural state
```

Comment:
May need to clarify that this doesn't mean it accurately represents the *current* architecture of the implementation. Rather it represents the current understanding of the *desired* architecture. The delta between current implementation state and desired architecture represents work that needs to be done to bring the implementation in line with the planned architecture.